### ISSUE TITLE
API function player.fullscreen.enter() doesn't work on Safari

### ISSUE THREAD

**Issue by edsardachuk** [sampotts/plyr#1681]
It is working on Chrome but on Safari I can go to fullscreen only by clicking on the 'fullscreen' button (inside controls). API function doesn't work for Safari. Same story with IE11.

**Comment by sampotts** [comment]
Any error in the console? It's usually due to requiring user input. I suspect that's the case here?

**Comment by edsardachuk** [comment]
Yes in Safari I have these 2 errors. I do not have them in the Chrome... https://take.ms/GVf1S

**Comment by edsardachuk** [comment]
I've added plyr.polyfilled.js.map file but it didn't help

**Comment by tbntdima** [comment]
any update on this?

**Comment by ffxsam** [comment]
@sampotts This is still an issue, any ideas on what the issue is? Or any potential workarounds?

**Comment by ffxsam** [comment]
BTW, I see in another issue that it won't work unless it's responding to an event binding. But in our case, when the user clicks the play button, we are calling `.fullscreen.enter()`. Not really sure why Safari is ignoring that click/touch event.
